code-quality/android-code-quality.gradle

```diff
@@ -0,0 +1,80 @@
+// wrapper task to run all static code analysis
+task staticAnalysis {
+    description 'Runs checkstyle, findbugs and pmd against the main sources.'
+    group 'Verification'
+}
+
+def sourceSetMain = android.sourceSets.main.java.srcDirs
+def rulesDir = new File("$project.rootDir/code-quality", 'rules')
+def ignoreFail = true // sad panda
+
+def includeSrcPatternApplication = 'com/novoda/downloadmanager/**/*.java'
+def excludeSrcPatternRClass = 'com/novoda/downloadmanager/**/R.java'
+def excludeSrcPatternGeneratedFiles = '**/gen/**'
+
+apply plugin: 'checkstyle'
+task checkstyleMain(type: Checkstyle) {
```

Comment:
can we still have these as 3 separate files so we can easily distinguish them and change them separately?